topics/statistics/tutorials/galaxy-ludwig/tutorial.md

```diff
@@ -0,0 +1,467 @@
+iii) The split configuration is used to divide the dataset based on the 'split' column rather than using a random split (which is the default). In this configuration, rows labeled with '0' are included in the training set, while rows labeled with '2' are included in the testing set. If a validation category is needed, rows are labeled with '1'. 
+iv) The model is trained to classify images into categories (e.g., recognizing digits), and it will train for 5 epochs.
+
+![Config.yaml file snapshot](../../images/galaxy-ludwig/explain_config.png "Config.yaml file snapshot.")
+
+# GALAXY-LUDWIG TOOL 
+
+Ludwig simplifies the complexities of machine learning by automating essential steps such as data preprocessing, model architecture selection, hyperparameter tuning, and device management. This streamlined approach is particularly beneficial for Galaxy users who are more interested in addressing their scientific questions than in navigating the intricacies of machine learning workflows.
+
+## Prepare environment and get the data 
+
+> <hands-on-title> Environment and Data Upload </hands-on-title>
+>
+> 1. Create a new history for this tutorial. If you are not inspired, you can name it *image classifier*.
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. Import the files from Zenodo or from the shared data library
+>
+>    ```
+>    https://zenodo.org/records/13292895/files/mnist_images.zip
+>    https://zenodo.org/records/13357260/files/mnist_dataset.csv 
+>    https://zenodo.org/records/13713095/files/config.yaml
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+> 3. Check that the data format assigned for each file is **zip**, **csv**, and **yaml**, respectively.
+>    If it is not, follow the Changing the datatype tip.
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
+>
+> 5. Add a tag (`Stacked CNN`) to the dataset corresponding to `config.yaml`
+>    This is important to trace back on what configurations the model was built on.
+>
+>    {% snippet faqs/galaxy/datasets_add_tag.md %}
+>
+{: .hands_on}
+
+## Ludwig Experiment
+
+> <hands-on-title> Task description </hands-on-title>
+>
+> 1. {% tool [Ludwig Experiment](ludwig_experiment) %} with the following parameters:
+>    - {% icon param-file %} *"Select the dataset containing model configuration"*: `config.yaml`
+>    - {% icon param-file %} *"Input dataset"*: `mnist_dataset.csv`
+>    - {% icon param-file %} *"Raw data"*: `mnist_images.zip`
+>
+>    > <comment-title> short description </comment-title>
+>    >
+>    > Once all the inputs are in place, press the 'RUN TOOL' button to start your job.
+>    {: .comment}
+>
+{: .hands_on}
+
+## Ludwig Experiment Outputs files
+
+After your model is trained and tested, you should see three new files in your history list:
+
+    > Ludwig Experiment Report: An HTML file containing the evaluation report of the trained model.
```

Comment:
I think this could look better as a markdown list rather than comment block

Reply:
Changed it to list.